rskj-core/src/main/java/co/rsk/scoring/PeerScoringBadReputationSummary.java

```diff
@@ -0,0 +1,149 @@
+package co.rsk.scoring;
+
+import java.util.Arrays;
+
+/**
+ * This is a presentational object of peers with bad reputation
+ * It's used to expose a json rpc message (sco_badReputationSummary())
+ */
+public class PeerScoringBadReputationSummary {
+    private int count;
+    private int successfulHandshakes;
+    private int failedHandshakes;
+    private int invalidNetworks;
+    private int repeatedMessages;
+    private int validBlocks;
+    private int validTransactions;
+    private int invalidBlocks;
+    private int invalidTransactions;
+    private int invalidMessages;
+    private int timeoutMessages;
+    private int unexpectedMessages;
+    private int invalidHeader;
+    private int peersTotalScore;
+    private int punishments;
+
+    public PeerScoringBadReputationSummary(int count,
```

Comment:
Why "BadReputation"? We could use this information for any peer, not only the ones with bad reputation. ie, the number of valid blocks

Reply:
[changed](https://github.com/rsksmart/rskj/pull/1351/commits/c2ad004ea2c9623122a60b38d69b45a3a99f52c6)